<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>openDPIdetectorGui</class>
 <widget class="QMainWindow" name="openDPIdetectorGui">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>306</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>769</width>
    <height>306</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>769</width>
    <height>306</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">Open DPI Detector</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>752</width>
      <height>233</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetNoConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="horizontalSpacing">
        <number>6</number>
       </property>
       <property name="verticalSpacing">
        <number>4</number>
       </property>
       <item row="1" column="0" alignment="Qt::AlignRight">
        <widget class="QLabel" name="dwFiLabel">
         <property name="text">
          <string>File:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="5">
        <widget class="QPushButton" name="wFiActionButton">
         <property name="minimumSize">
          <size>
           <width>160</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Save DPI of file</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" alignment="Qt::AlignRight">
        <widget class="QLabel" name="wFiLabel">
         <property name="text">
          <string>File:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="wFiLineEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0" alignment="Qt::AlignRight">
        <widget class="QLabel" name="dFiLabel">
         <property name="text">
          <string>File:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="3" alignment="Qt::AlignRight">
        <widget class="QLabel" name="wFiDpiLabel">
         <property name="text">
          <string>DPI:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="4">
        <widget class="QComboBox" name="dwFoOrientationComboBox">
         <property name="minimumSize">
          <size>
           <width>65</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="whatsThis">
          <string notr="true"/>
         </property>
         <item>
          <property name="text">
           <string>up</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>right</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>down</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>left</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="dwFoLineEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="dwFoChooseButton">
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Choose</string>
         </property>
        </widget>
       </item>
       <item row="3" column="5">
        <widget class="QPushButton" name="dwFoActionButton">
         <property name="minimumSize">
          <size>
           <width>160</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Detect and save DPI of folder</string>
         </property>
        </widget>
       </item>
       <item row="1" column="5">
        <widget class="QPushButton" name="dwFiActionButton">
         <property name="minimumSize">
          <size>
           <width>160</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Detect and save DPI of file</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QComboBox" name="dwFiOrientationComboBox">
         <property name="minimumSize">
          <size>
           <width>65</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="whatsThis">
          <string notr="true"/>
         </property>
         <item>
          <property name="text">
           <string>up</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>right</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>down</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>left</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="3" alignment="Qt::AlignRight">
        <widget class="QLabel" name="wFoDpiLabel">
         <property name="text">
          <string>DPI:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="wFiChooseButton">
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Choose</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3" alignment="Qt::AlignRight">
        <widget class="QLabel" name="dwFoOrientationLabel">
         <property name="text">
          <string>Mark orientation:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QPushButton" name="dFiActionButton">
         <property name="minimumSize">
          <size>
           <width>160</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Detect DPI of file</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QPushButton" name="wFoChooseButton">
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Choose</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="wFoLineEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="3" alignment="Qt::AlignRight">
        <widget class="QLabel" name="dwFiOrientationLabel">
         <property name="text">
          <string>Mark orientation:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="5">
        <widget class="QPushButton" name="wFoActionButton">
         <property name="minimumSize">
          <size>
           <width>160</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Save DPI of folder</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" alignment="Qt::AlignRight">
        <widget class="QLabel" name="wFoLabel">
         <property name="text">
          <string>Folder:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="dFiChooseButton">
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Choose</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3" alignment="Qt::AlignRight">
        <widget class="QLabel" name="dFiOrientationLabel">
         <property name="text">
          <string>Mark orientation:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QComboBox" name="dFiOrientationComboBox">
         <property name="minimumSize">
          <size>
           <width>65</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="whatsThis">
          <string notr="true"/>
         </property>
         <item>
          <property name="text">
           <string>up</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>right</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>down</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>left</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="dwFiLineEdit">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="dwFiChooseButton">
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Choose</string>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QSpinBox" name="wFiDpiSpinBox">
         <property name="whatsThis">
          <string notr="true"/>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>65535</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="4" column="4">
        <widget class="QSpinBox" name="wFoDpiSpinBox">
         <property name="whatsThis">
          <string notr="true"/>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>65535</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="dFiLineEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0" alignment="Qt::AlignRight">
        <widget class="QLabel" name="dwFoLabel">
         <property name="text">
          <string>Folder:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="1">
        <widget class="QTextBrowser" name="outputTextBrowser">
         <property name="undoRedoEnabled">
          <bool>false</bool>
         </property>
         <property name="html">
          <string notr="true">&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="acceptRichText">
          <bool>false</bool>
         </property>
         <property name="cursorWidth">
          <number>3</number>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::TextSelectableByMouse</set>
         </property>
         <property name="openLinks">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="forceCancelButton">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Force stop</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="cancelButton">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>769</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLanguage">
    <property name="title">
     <string>Language</string>
    </property>
   </widget>
   <addaction name="menuLanguage"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
